ci/test/03_test_script.sh

```diff
@@ -234,6 +240,13 @@ if [ "${RUN_TIDY}" = "true" ]; then
   run_iwyu "compile_commands_iwyu_errors.json"
   if ! ( git --no-pager diff --exit-code ); then
     echo "^^^ ⚠️ Failure generated from IWYU"
+    echo
+    echo "Some adjustments to the diff may be needed:"
+    echo
+    echo "* Use the C++ headers: E.g. <cerrno> over <errno.h>"
+    echo '* Use #include <__.h> over #include "__.h" for all headers'
```

Comment:
I'm wondering, give we are building IWYU from source, can we just patch it, so the output is correct (uses <>); avoiding the need for this fixup.